packages/cli-kit/src/private/node/conf-store.ts

```diff
@@ -77,7 +78,7 @@ export async function cacheRetrieveOrRepopulate(
   const cache: Cache = config.get('cache') || {}
   const cached = cache[key]
 
-  if (cached && (timeout === undefined || Date.now() - cached.timestamp < timeout)) {
+  if (cached?.value && (timeout === undefined || Date.now() - cached.timestamp < timeout)) {
```

Comment:
This won't work if the cached value is `false`.

Reply:
This is my point. If `cached.value === false` it will treat it as uncached.

Reply:
Oh, right! 🤦